addons/website/static/src/scss/website.scss

```diff
@@ -863,11 +863,16 @@ header {
     #top_menu_collapse {
 
         &.collapsing, &.show {
+            // Note: position relatively to the header instead of the viewport
+            // because fixed position cannot work inside an element whose CSS
+            // transform is different to none, which the header element is
+            // because of header effects.
             @include o-position-absolute(0, 0, 0, 0);
-            position: fixed;
             z-index: $zindex-sticky;
+            height: 100vh;
             transition: none;
             transform: none;
+            overflow-x: hidden;
```

Comment:
@bvr-odoo 
> Im not sure about this fix, it works but it's a workaround ... 

I think this is a good fix considering the problem :+1: I rebased, fixed the conflicts then made this small change: https://github.com/odoo/odoo/compare/ec4c076019af344e303271c630e2f45b3315cc85..c86758548ef8986458801305f03c5151e0171097 (explaining the workaround and avoiding the use of `calc`).

However, I am not sure the overflow-x line I am commenting here is actually needed, do you know why you added it?

Reply:
It was to avoid to have the horizontal scrollbar during the animation of the off-canvas menu (not affixed) when its appears. But with the new right panel we don't need it anymore so you can delete this line :+1: